can_bus/quintuna/BRUSA/BRUSA_tx.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
I think that some of the messages from the BRUSA were being used in the BMS for state and charging control. Unless the whole message of like 32 values needs to be defined though for it to show up legibly on PCAN (pretty sure) which might be why it wasn't included. I can just add those messages to master, test, and then add it separately, but I think the three messages that were there should be included.

Reply:
I just added the CAN messages so that it doesn't error out when I try to build. This configuration is completely up to you.